endpoints.json

```diff
@@ -119,5 +119,22 @@
       }
     ]
   }
+},
+"GET /api/articles/:article_id(comment_count)" :{
```

Comment:
this is the same endpoint as before so you should just update your previous endpoint